BenchmarkTests/SwiftUICatalog/ContentView.swift

```diff
@@ -0,0 +1,345 @@
+                     label: "Gestures")
+                Link(destination: ComposingGesturesView().modifier(monitor.viewModifier(name: "Composing Gestures")),
+                     label: "Composing Gestures")
+                Link(destination: SensoryFeedbackInViews().modifier(monitor.viewModifier(name: "Sensory Feedback")),
+                     label: "Sensory Feedback")
+            }
+            .listRowBackground(Color(sectionColor, bundle: .module))
+
+    }
+    
+    var viewModifiers: some View {
+        Section(header: Text("View modifiers")
+            .font(.title)
+            .modifier(ListSectionFontModifier())) {
+                Link(destination: TextModifiersView().modifier(monitor.viewModifier(name: "Text modifiers")),
+                     label: "Text modifiers")
+                Link(destination: EffectsModifiersView().modifier(monitor.viewModifier(name: "Effect modifiers")),
+                     label: "Effect modifiers")
+                Link(destination: LayoutModifiersView().modifier(monitor.viewModifier(name: "Layout modifiers")),
+                     label: "Layout modifiers")
+                
+            }
+            .listRowBackground(Color(sectionColor, bundle: .module))
+
+    }
+    
+    var accessibility: some View {
+        Section(header: Text("Accesibility")
+            .font(.title)
+            .modifier(ListSectionFontModifier())) {
+                
+                Link(destination: AccesibilityView().modifier(monitor.viewModifier(name: "Accesibility")),
+                     label: "Accesibility")
+            }
+            .listRowBackground(Color(sectionColor, bundle: .module))
+
+    }
+    
+    var statusBars: some View {
+        Section(header: Text("Status and tool bars")
+            .font(.title)
+            .modifier(ListSectionFontModifier())) {
+                Link(destination: ToolbarsComponentView().modifier(monitor.viewModifier(name: "Tool Bars")),
+                     label: "Tool Bars")
+            }
+            .listRowBackground(Color(sectionColor, bundle: .module))
+
+    }
+    
+    var stylingSection: some View {
+        Section(header: Text("Styling")
+            .font(.title)
+            .modifier(ListSectionFontModifier())) {
+                
+                Link(destination: StylesView().modifier(monitor.viewModifier(name: "Styles")),
+                     label: "Styles")
+            }
+            .listRowBackground(Color(sectionColor, bundle: .module))
+
+    }
+    
+    var popovers: some View {
+        Section(header: Text("Popovers, alerts and sheets")
+            .font(.title)
+            .modifier(ListSectionFontModifier())) {
+                
+//                Link(destination: PopoversComponentView(),
+//                     label: "Popovers")
+                Link(destination: SheetsView().modifier(monitor.viewModifier(name: "Sheets")),
+                     label: "Sheets")
+//                Link(destination: AlertsComponentView(),
```

Comment:
question/ Do you want the links hidden? A configuration with a toggle might help here

Reply:
They are not actually implemented, we can keep them hidden and add them later 👍